### PR TITLE
Fix docs for data invariant & remove tuple header

### PR DESCRIPTION
**Pull request by sonodtt**
Inconsistency in final "summary statement" for data invariant
• TLDR:
For data-invariant the statement was:
"The data structure is corrupt or malformed"
Amended this to:
"The data structure is not corrupt or malformed"

Also - header no longer needed.

More info:
e.g
• precondition
"The design of the system means that the arguments to this method
will always meet this condition".
(positive assertion)
• postcondition
"The implementation of this method means that the condition will hold".
(positive assertion)
• check-return
"The contract of the previous method call means the following condition holds".
(positive assertion)

For data-invariant the statement was:
"The data structure is corrupt or malformed"
(incorrect negative assertion)

Amended this to:
"The data structure is not corrupt or malformed"

### CODE REVIEW

File: src/util/invariant.h [hunk 1]
```diff
@@ -12,7 +12,6 @@ Author: Martin Brain, martin.brain@diffblue.com
 #include <cstdlib>
 #include <stdexcept>
 #include <string>
-#include <tuple>
```
Comment (tautschnig) [inline src/util/invariant.h:15]: Procedural nit-pick: do this in a separate commit. Possibly even a separate pull request.

**Review by tautschnig — APPROVED** [review]

**Review by hannes-steffenhagen-diffblue — APPROVED** [review]

File: src/util/invariant.h (hunk 1 above)
Comment (sonodtt) [reply src/util/invariant.h:15]: Done. In future - separate PR.
